### PR TITLE
Winbuild: Use junctions in MSYS too

### PR DESCRIPTION
**Pull request by bagong**
Fix for "symlinking" class- 'n help-folders in build target-folders.

### CODE REVIEW

File: platform/windows/junctions.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# USE: symlinks.sh create <target> <link>
+# USE: symlinks.sh remove <link>
```
Comment (brianlheim) [inline platform/windows/junctions.sh]: This needs to be renamed too.

File: platform/windows/junctions.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# For both, link and target, the parent folder of the folder group SCClassLibrary, HelpSource, examples and sounds has to be given
+# rm -r does *not* remove the files contained in the target dir ;). This seems to be a peculiarity of msys2 symlink support
```
Comment (brianlheim) [inline platform/windows/junctions.sh]: Can we remove the `;)` here while we're at it?

File: platform/windows/junctions.sh (hunk 1 above)
Comment (bagong) [inline platform/windows/junctions.sh]: done

File: platform/windows/junctions.sh (hunk 2 above)
Comment (bagong) [inline platform/windows/junctions.sh]: if it makes you smile ;)

**Pull request merged**